zipline/gens/tradesimulation.py

```diff
@@ -189,6 +189,10 @@ def inner_loop(dt_to_use, orders_to_process):
                         all(not order.open for order in remaining_open_orders):
                     del open_orders[sid]
 
+            # update the portfolio, so that if the user does
+            # context.portfolio.positions, it's accurate
+            perf_tracker.get_portfolio(True)
```

Comment:
This is a good stopgap, which should be removed/obviated when I have portfolio.positions working fully.